Sprint1/src/main/java/App.java

```diff
@@ -8,9 +8,15 @@
 
 	public static void main(String[] args) {
 		// TODO: Change this later, for now we can just call getInstance on each manager to run the parsers
-		UserManager.getInstance();
-		TheatreManager.getInstance();
-		BookingManager.getInstance();
+		UserManager user = new UserManager();
```

Comment:
Same thing with these lines, we need to use the getInstance() functions to get the current static instance of each class. I will explain in the next meeting.